devtools/enforcer-rules/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-devtools-all</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-enforcer-rules</artifactId>
+    <name>Quarkus - Enforcer Rules</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.maven.enforcer</groupId>
+            <artifactId>enforcer-api</artifactId>
+            <version>${version.enforcer.plugin}</version>
+            <scope>provided</scope>
+            <!-- avoid dependencyConvergence issues -->
+            <exclusions>
+                <exclusion>
+                    <groupId>org.eclipse.sisu</groupId>
+                    <artifactId>org.eclipse.sisu.plexus</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-core</artifactId>
+            <scope>provided</scope>
+            <!-- avoid dependencyConvergence issues -->
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-invoker-plugin</artifactId>
+                <version>3.2.1</version>
```

Comment:
Care to put this version string into a property? There's a couple of other modules that use Maven Invoker plugin, and perhaps we should centralize its version. Can be an extra PR too, I don't mind.

Reply:
Good point. I actually had a look first whether other modules are using this plugin and back then I found none.

But there are two "maven-invoker-way" submodules under `integration-tests` that are using this plugin.

Extra PR sounds reasonable since the version in those two modules are already inconsistent.

Reply:
Agree.

Reply:
As a reminder I'll leave this open until this PR is merged.